src/main/java/InputStreamReader.java

```diff
@@ -0,0 +1,29 @@
+import org.apache.commons.io.IOUtils;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class InputStreamReader {
+    private InputStream inputStream;
+
+    public InputStreamReader(InputStream inputStream) {
+        this.inputStream = inputStream;
+    }
+
+    /**
+     * Method to read from inputStream and return a byte array of all bytes read
+     * Uses ByteArrayOutputStream to construct byte array which will be returned
+     */
+    public byte[] readStream() {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
don't save space on variable names. make them as long as you need to and as readable as you can